build.gradle

```diff
@@ -237,6 +237,21 @@ project(':iceberg-data') {
   }
 }
 
+project(':iceberg-aws') {
+  dependencies {
+    compile project(':iceberg-api')
+    compile project(':iceberg-core')
+
+    compileOnly "com.amazonaws:aws-java-sdk-core"
```

Comment:
Are you sure that this is needed?

I was able to open this PR, remove this line, run `./gradlew clean build`, and everything passed just fine for me. 

All of your imports are from `com.amazonaws.services.s3.*` so I really don't think that core is needed, but gradle isn't my primary build tool at work so maybe I'm missing something.

Reply:
Good point.  Core is actually a dependency of S3 anyway, so it is pulled transitively.  No need to depend on it directly.